src/components/Badge/Badge.tsx

```diff
@@ -1,41 +1,49 @@
 import React from "react";
-import "./statuswithbadge.scss"
+import "./badge.scss";
+import Ellipse from "../Ellipse";
 
-export interface StatusWithBadgeProps {
+export interface BadgeProps {
     status:String    
   }
 
-  const StatusWithBadge=({status}:StatusWithBadgeProps)=>{
+  const Badge=({status}:BadgeProps)=>{
     const state=(():any=>{
         switch (status) {
             case "live":                
                 return {
                     className:"swb-live",
+                    color:"#5EAB6C",
                     title:"LIVE",
               };
             case "recording":
                 return {
                     className:"swb-recording",
+                    color:"#943734",
                     title:"LIVE",
+                    blink:true
                 }
             case "offline":
                 return {
                   className:"swb-offline",
+                  color:"#6D6666",
                   title:"OFFLINE",
                 }
             case "event-detected":
               return {
                   className:"swb-event-detected",
+                  color:"#943734",
                   title:"EVENT DETECTED",
               }
             case "active":
               return {
                 className:"swb-active",
+                color:"#5EAB6C",
                 title:"ACTIVE",
               }    
             case "update":
               return {
                 className:"swb-update",
+                color:"#DFB211",
```

Comment:
Instead of passing those colors in I believe we should use the variables we used in the .app.variables.scss.
I understand what you try to do, but I think we should build the ellepsise,so that it's can recognize color names e.g..

`<Ellipse status="alert"/>`

Instead of 

`<Ellipse color="#fff"/>`

The last will give complications if we will change the colours.

Reply:
Now it's ok both Badge and Ellipse components. Now they accept status as variables defined in app.variables.scss. But since recording status for badge needs a different background I also put it.